usecases/scenario_iterations_usecase.go

```diff
@@ -119,9 +119,14 @@ func (usecase *ScenarioIterationUsecase) CreateScenarioIteration(ctx context.Con
 		body.ScoreReviewThreshold = &defaultReviewThreshold
 	}
 
-	if body.ScoreRejectThreshold == nil {
-		defaultRejectThreshold := 10
-		body.ScoreRejectThreshold = &defaultRejectThreshold
+	defaultRejectThreshold := 10
+	if body.ScoreBlockAndReviewThreshold == nil {
+		// the block and review outcome cannot be reached with the default scenario iteration, as backward compatibility
+		body.ScoreBlockAndReviewThreshold = &defaultRejectThreshold
+	}
```

Comment:
If `ScoreDeclineThreshold != nil` it will create space for this threshold. But assuming the frontend only create scenario with defaults, we can safelly ignore.

In fact, the problem of "partial thresholds provided at creation" raise a lot of complexity if we want to ensure validity of each default threshold.

Examples:
1.  `ScoreReviewThreshold = 15`
2. `ScoreDeclineThreshold < 10`
3. `ScoreDeclineThreshold > 10`
4. `ScoreBlockAndReviewThreshold > 10, ScoreDeclineThreshold = nil`
5. ...

Reply:
No it's alright, though my comment was maybe misleading. The default scenario iteration is only used when a new scenario is created.
Technically by API you can create a non-empty iteration, and it can be invalid, but that is fine - the control is done at commit time. (just like you can set inconsistent thresholds by API, but you can't commit them AFAIR)

Reply:
the "as backward compatibility" is useless however

Reply:
what I won't do now is rename the field in DB